fdbserver/BackupWorker.actor.cpp

```diff
@@ -250,69 +435,92 @@ ACTOR Future<Void> monitorBackupStartedKeyChanges(BackupData* self, bool started
 	}
 }
 
-// Monitor all backup worker in the recruited epoch has been started. If so,
-// set the "allWorkerStarted" key of the BackupConfig to true, which in turn
-// unblocks StartFullBackupTaskFunc::_execute. Note only worker with Tag (-2,0)
-// runs this actor so that the key is set by one process.
-ACTOR Future<Void> monitorAllWorkerStarted(BackupData* self) {
+// Set "latestBackupWorkerSavedVersion" key for backups
+ACTOR Future<Void> setBackupKeys(BackupData* self, std::map<UID, Version> savedLogVersions) {
 	loop {
-		wait(delay(SERVER_KNOBS->WORKER_LOGGING_INTERVAL / 2.0) || self->changedTrigger.onTrigger());
-		if (self->backups.empty()) {
-			continue;
-		}
+		state Reference<ReadYourWritesTransaction> tr(new ReadYourWritesTransaction(self->cx));
```

Comment:
move out of loop

Reply:
Add PRIORITY_SYSTEM_IMMEDIATE to all transactions in backup worker

Reply:
Done.